doc/source/release-notes/Percona-Server-5.7.23-24.rst

```diff
@@ -0,0 +1,45 @@
+.. rn:: 5.7.23-24
+
+================================================================================
+Percona Server 5.7.23-24
+================================================================================
+
+Percona announces the release of `Percona Server for MySQL
+<https://www.percona.com/software/percona-server>`_ 5.7.23-24 on November 9,
+2018 (downloads are available `here <https://www.percona.com/downloads/Percona-Server-5.7/>`_ and from the `Percona Software Repositories <https://www.percona.com/doc/percona-server/5.7/installation.html#installing-from-binaries>`_). This
+release merges changes of MySQL 5.7.24, including all the bug fixes in
+it. Percona Server for MySQL 5.7.23-24 is now the current GA release in the 5.7
+series. All of Percona’s software is open-source and free.
+
+This release introduces InnoDB encryption improvements and merges upstream
+MyRocks changes. Also, the usage of column families in MyRocks has been
+improved. The InnoDB encryption improvements are in **Alpha** quality and are
+not recommended to be used in production.
+
+New Features
+================================================================================
+
+- :psbug:`4905`: Upstream MyRocks changes have been merged
+- :psbug:`4976`: InnoDB Undo Log Encryption has been implemented
+- :psbug:`4946`: Add the ``rocksdb_no_create_column_family`` option to prevent the implicit creation of column families in MyRocks
+- :psbug:`4556`: InnoDB :ref:`ps.data-at-rest-encryption.redo-log` has been implemented
+- :psbug:`3839`: InnoDB Data Scrubbing has been implemented
```

Comment:
PS-3834?

Reply:
PS-3829, and any mention of keyring encryption is also missing from here. Unfortunately PS-3829 is set to internal, so it isn't publicly visible. But we should mention keyring encryption here.

Reply:
Let's go with PS-3829 even if not public. We can make it public later

Reply:
Issue added